guide-us-english.tex

```diff
@@ -439,7 +439,7 @@ \section{End Note}
 
 Scrum@Scale is designed to saturate an organization with Scrum. All teams, including Leadership, Human Resources, Legal, Consulting \& Training, and product \& service teams, implement the same style of Scrum while streamlining and enhancing an organization.
 
-Well implemented Scrum can run an entire organization.
+Well implemented Scrum@Scale can run an entire organization.
```

Comment:
Neutral on this one. I could make a case for just "Scrum" as well, if it's intended to mean the whole Scrum ecosystem.